assembl/static2/js/app/reducers/rootReducer.js

```diff
@@ -17,4 +17,9 @@ export default combineReducers({
   auth: Auth,
   phase: Phase,
   admin: Admin
-});
\ No newline at end of file
+});
+
+// TODO use a proper key in redux store and create redux actions to set it
+export const getContentLocale = (state) => {
```

Comment:
Oh, this isn't a key in redux? There will be a key in redux store as an array of discussion pref'ed languages. Will this translation feature allow translated content to be in locale's that are not supported by the interface (ie anything but french, english, japanese)?

Reply:
Yes, for this intermediate version, the content locale is actually the same as the interface locale. In the final version this will be a separate key in redux so you will be able to translate into any language.